src/libraries/System.Net.WebSockets.Client/tests/CloseTest.cs

```diff
@@ -362,6 +362,38 @@ public async Task CloseOutputAsync_ServerInitiated_CanSend(Uri server)
             }
         }
 
+
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/28957", typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
+        [OuterLoop("Uses external servers", typeof(PlatformDetection), nameof(PlatformDetection.LocalEchoServerIsNotAvailable))]
+        [ConditionalTheory(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
+        public async Task CloseOutputAsync_ServerInitiated_CanSendRepeatedly(Uri server)
+        {
+            using (ClientWebSocket cws = await GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
+            {
+                var cts = new CancellationTokenSource(TimeOutMilliseconds);
+
+                await cws.SendAsync(
+                    WebSocketData.GetBufferFromText(".shutdownAfterTwoMessages"),
+                    WebSocketMessageType.Text,
+                    true,
+                    cts.Token);
+
+                var recvBuffer = new ArraySegment<byte>(new byte[1024]);
+                WebSocketReceiveResult recvResult = await cws.ReceiveAsync(recvBuffer, cts.Token);
+                var message = Encoding.UTF8.GetString(recvBuffer.ToArray(), 0, recvResult.Count);
+                Console.WriteLine($"Received: {message}");
+                Assert.Equal(WebSocketState.Open, cws.State);
```

Comment:
we should not assert the `State` here, we don't mind if it's closed already. We should assert the message content.